reliabot/reliabot.py

```diff
@@ -41,10 +41,21 @@
 from typing import TextIO
 from typing import Union
 
-from ruamel.yaml import YAML  # ruamel.yaml preserves comments, PyYAML doesn't.
-from ruamel.yaml.comments import CommentedMap
-from ruamel.yaml.comments import CommentedSeq
-from ruamel.yaml.parser import ParserError
+try:
+    # ruamel.yaml preserves comments, PyYAML doesn't.
+    from ruamel.yaml import YAML
+    from ruamel.yaml.comments import CommentedMap
+    from ruamel.yaml.comments import CommentedSeq
+    from ruamel.yaml.parser import ParserError
+except ModuleNotFoundError, ImportError as e:
+    error_message = ''.join([
+        f"{e}",
+        "\n\n",
+        "Reliabot requires the ruamel.yaml module to preserve comments in dependabot.yml files.",
+        "\n",
+        "See https://github.com/dupuy/reliabot/#installation for installation instructions."
+    ])
+    sys.exit(error_message)
```

Comment:
The `sys.exit()` function takes an integer exit code as its argument and doesn't print anything.

The `error(message: str) -> None` function below that will "Report a bug (internal error) in reliabot and exit with status 9." would be a better choice. To use it, the import statements have to be moved after the `error()` function definition.

Reply:
In PR #20 (for the similar issue #16) I also ended up moving the re2 import down to the top of the "constant" declaration section.

At this point this PR is also a few commits back from the main branch, so I'll try to update the branch so that there's not as much to clean up.

Reply:
Fixed this as well, note that I had to move the import handling even further to account for `DOCTEST_OPT` not being defined - I think a fun task would be to refactor the OPT constants into a separate file, however I am not sure if that is pythonic or a best practice

Reply:
Unresolving in case you want to re-review this